src/cargo/ops/common_for_install_and_uninstall.rs

```diff
@@ -545,11 +546,14 @@ impl InstallInfo {
 
 /// Determines the root directory where installation is done.
 pub fn resolve_root(flag: Option<&str>, gctx: &GlobalContext) -> CargoResult<Filesystem> {
-    let config_root = gctx.get_path("install.root")?;
```

Comment:
nit: shouldn't we remove `GlobalContext::get_path` entirely in the same commit?

https://github.com/rust-lang/cargo/issues/16023#issuecomment-3408768620

Reply:
Oops, forgot to remove it.

Reply:
Removed!